lib/justice_gov_sk/infrastructure.rb

```diff
@@ -0,0 +1,19 @@
+module JusticeGovSk
+  module Infrastructure
+    # TODO implement async?
+
+    def crawl
+      (1..pages).each do |page|
+        url = uri.build(page: page)
```

Comment:
ok, ja len ci si to nazval `url` prave pre to, co si teraz napisal, alebo preto, ze `uri` uz bolo obsadene? :D